.ado/templates/apple-xcode-build.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
+  xcode_sdk: ''
+  xcode_configuration: ''
+  xcode_workspacePath: ''
+  xcode_scheme: ''
+  xcode_actions: ''
+  xcode_useXcpretty: false
+  xcode_extraArgs: ''
+
+steps:
+  - task: Xcode@5
+    displayName: 'Xcode ${{ parameters.xcode_actions }} ${{ parameters.xcode_configuration }} ${{ parameters.xcode_sdk }} ${{ parameters.xcode_scheme }}'
+    inputs:
+      actions: '${{ parameters.xcode_actions }}'
+      configuration: ${{ parameters.xcode_configuration }}
+      sdk: ${{ parameters.xcode_sdk }}
+      xcWorkspacePath: ${{ parameters.xcode_workspacePath }}
+      scheme: ${{ parameters.xcode_scheme }}
+      xcodeDeveloperDir: /Applications/Xcode_11.2.1.app/Contents/Developer
```

Comment:
Is there a way to extract the xcode version? Not sure if this is the only place we specify this, but it doesn't seem intuitive for someone to go easily update when we get to 11.4

Reply:
Update - it's not the only place. Just saw the apple-xcode-select script

Reply:
This logic is now centralized in scripts/xcode_select_current_version.sh for both GitHub Actions CI and the new ADO NuGet publishing pipeline.